t/local/back.t

```diff
@@ -3,7 +3,7 @@ use strict;
 use Test::More tests => 47;
 use lib qw( t t/local );
 use LocalServer;
-use HTTP::Daemon;
+use HTTP::Daemon 6.05;
```

Comment:
Or, we could also add this to TestRequires in `dist.ini`: https://github.com/libwww-perl/WWW-Mechanize/blob/master/dist.ini#L26-L27

Reply:
Currently the prereq version for HTTP::Daemon is 0, but has be set to 6.05, otherwise tests will fail if older HTTP::Daemon is installed. Or alternatively skip tests if HTTP::Daemon version is < 6.05.

Reply:
As I mentioned in the first comment, `dzil build` will bump HTTP::Daemon prereq.
I intentionally did not do it because it would also give us unrelated diff.

Reply:
I manually bumped HTTP::Daemon in 68af39e